### PR TITLE
Modify DataServlet to include JSON

### PR DESCRIPTION
**Pull request by lornaarmstrong**
to contain an ArrayList<String> and doGet() returns the ArrayList as JSON string.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -24,9 +27,26 @@
...
+      "\"Le monde est un livre dont chaque pas nous ouvre une page.\""
+      + " -- Alphonse de Lamartine"
+      + "\n *** Translation: The world is a book - with every step, we open a page. ***");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    //return the ArrayList as JSON string
+    String json = new Gson().toJson(favQuotes);
     response.setContentType("text/html;");
```
Comment (MostafaGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: The content type is no longer an html text :) Please update it to `application/json`
[JavaDoc](https://docs.oracle.com/javaee/6/api/javax/servlet/ServletResponse.html#setContentType(java.lang.String))
[Common MIME types](http://go/mdn/HTTP/Basics_of_HTTP/MIME_types/Common_types)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -12,26 +12,11 @@
...
-*/
-
-
 /**
  * Fetches greeting from the server and adds it to the DOM.
  */
-function fetchAndShowGreeting() {
-  console.log('Fetching a greeting.');
+function fetchAndShowFavQuote() {
+  console.log('Fetching a favourite quote.');
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/script.js:19]: nitty nit: `quotes` :) same below

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -223,3 +223,7 @@ footer {
 .extra-border-img{
   margin-top:12px;
 }
+
+#quote-container{
+    margin: 10px;
+}
```
Comment (MostafaGabriel) [inline portfolio/src/main/webapp/style.css:229]: please add a new line at the of the file :) you can also delete the greeting-container as it's no longer being used

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (KaziZubaer) [reply portfolio/src/main/webapp/script.js:19]: Actually +1 to that :D I got a bit confused thinking if one quote will be shown or all three of them will be shown.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (lornaarmstrong) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: Thanks!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/style.css:229]: Thanks, good spot!

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,26 +12,11 @@
...
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-*/
-
-
 /**
  * Fetches greeting from the server and adds it to the DOM.
```
[the remark this answers is not in the document]
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/script.js:16]: Will do, thanks!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (lornaarmstrong) [reply portfolio/src/main/webapp/script.js:19]: Good point, thanks!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -23,30 +23,25 @@
...
+    comments.add("It seems like a really cool portfolio.");
+    comments.add("I like swing dance too!");
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //return the ArrayList as JSON string
-    String json = new Gson().toJson(favQuotes);
-    response.setContentType("text/html;");
+    Gson gson = new Gson();
```
Comment (MostafaGabriel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: nit: It's better to have this inside `init` to only have one instance of the object

Comment (lornaarmstrong) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: Thanks Mostafa!

**Pull request merged**